src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,11 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: $normal-font-weight;
+  font-size: $normal-font-size;
+  line-height: $normal-line-height;
+  color: var(--text-secondary-color);
```

Comment:
I think you forgot to change this one :)

Reply:
How? I don't understand

Reply:
@MabuzaM He probably means that your `color` variable is in the CSS format, but not SCSS. But I don't think it's a mistake tbh.